src/main/java/org/zalando/planb/revocation/persistence/CassandraAuthorizationRuleStore.java

```diff
@@ -0,0 +1,112 @@
+package org.zalando.planb.revocation.persistence;
+
+import com.datastax.driver.core.BoundStatement;
+import com.datastax.driver.core.ConsistencyLevel;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.RegularStatement;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.querybuilder.QueryBuilder;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.zalando.planb.revocation.domain.AuthorizationRule;
+import org.zalando.planb.revocation.domain.ImmutableAuthorizationRule;
+
+import javax.annotation.PostConstruct;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static com.datastax.driver.core.querybuilder.QueryBuilder.bindMarker;
+import static com.datastax.driver.core.querybuilder.QueryBuilder.now;
+
+@Slf4j
+public class CassandraAuthorizationRuleStore implements AuthorizationRulesStore.Internal {
+
+    private static final long RELOAD_IN_MS =  1000 * 60 * 10; // 10 minutes
```

Comment:
No hardcoded values - this should be configurable - [here](https://github.com/zalando/planb-revocation/blob/master/src/main/java/org/zalando/planb/revocation/config/properties/CassandraProperties.java), for instance. Also, switch it to seconds, since it's the unit for all time related configs (not only here but in planb-provider, for example).